tools/generator/src/Generator/DisambiguateTargets.swift

```diff
@@ -41,10 +41,8 @@ extension Generator {
         }
 
         // And then distinguish them
-        var uniqueValues = Dictionary<
-            ConsolidatedTarget.Key,
-            DisambiguatedTarget
-        >(minimumCapacity: targets.count)
+        var uniqueValues = [ConsolidatedTarget.Key:
+            DisambiguatedTarget](minimumCapacity: targets.count)
```

Comment:
```suggestion
        var uniqueValues = [
            ConsolidatedTarget.Key: DisambiguatedTarget
        ](minimumCapacity: targets.count)
```

Reply:
Ohh, if it accepts a trailing comma there, add that too. I was mainly calling out the way the newline was being mis-used.

Reply:
I was able to remove the directive and add a comma.